tests/unit/form-specs.tsx

```diff
@@ -25,4 +40,27 @@ describe("<Form/>", () => {
 
         expect(isDefaultPrevented).to.be.true;
     });
+
+    it("should set classes for ReCAPTCHA on submit", () => {
+
+        wrapper.simulate("submit");
+
+        expect(document.body.className).to.contain("recaptcha-check");
+
+        expect(document.body.querySelector(".iframe-wrap")).to.exist;
+        expect(document.body.querySelector(".recaptcha-modal")).to.exist;
+
+        expect((node as any).recaptchaWrap).to.exist;
```

Comment:
recaptchaWrap is protected property. Should it be tested? Why? Tests must test public interface, not internal.